kafka_consumer/datadog_checks/kafka_consumer/client/kafka_client.py

```diff
@@ -5,17 +5,8 @@
 
 
 class KafkaClient(ABC):
-    def __init__(self, check, tls_context) -> None:
-        self.check = check
```

Comment:
To reviewer: I don't think we need to define the instance variables in this parent class.